mne/viz/evoked.py

```diff
@@ -2046,7 +2047,9 @@ def plot_compare_evokeds(evokeds, picks=None, gfp=False, colors=None,
         callable, it must take as its single argument an array
         (observations x times) and return the upper and lower confidence bands.
         If None or False, no confidence band is plotted.
-        If True, a 95%% bootstrapped confidence interval is drawn.
+        If True, a 95%% bootstrapped confidence interval is drawn, unless
+        ``evokeds`` does not have > 2 Evoked objects for each condition,
+        in which case no CI is drawn.
```

Comment:
if there is a decision logic then I prefer to use 'auto' as if I pass True I don't expect some logic to change this to False.